src/openrct2/world/Park.cpp

```diff
@@ -79,17 +77,6 @@ int32_t _suggestedGuestMaximum;
  */
 int32_t _guestGenerationProbability;
 
-/**
- *
- *  rct2: 0x00667104
- */
-void reset_park_entry()
```

Comment:
Why did you remove this?

Reply:
It was only called from one place which already initialised the data.